imod/mf6/riv.py

```diff
@@ -319,6 +331,9 @@ def from_imod5_data(
         )
         regridded_package_data["conductance"] = river_conductance
         regridded_package_data.pop("infiltration_factor")
+        regridded_package_data["bottom_elevation"] = regridded_package_data[
+            "bottom_elevation"
+        ].transpose("time", "layer", "y", "x", missing_dims="ignore")
```

Comment:
now using enforce_dim_order. In the test case, without this rearrangement of the data the test fails because the coordinates where not in the required order. Specifically, in the test using the periodic dataset,  Bottom_Elevation has a time coordinate at the end (and x, y instead of y, x):
```
imod5_data[key]["bottom_elevation"]  has
Coordinates:
  * x        (x) float64 1kB 1.947e+05 1.947e+05 ... 1.993e+05 1.993e+05
  * y        (y) float64 1kB 3.653e+05 3.653e+05 ... 3.607e+05 3.607e+05
    dx       float64 8B 25.0
    dy       float64 8B -25.0
  * time     (time) datetime64[ns] 16B 1899-04-01 1899-10-01
```
The extra time coordinate is apparently produced by the `open_projectfile_data` method. Since we expect time, layer, y, x
the test would fail in the constructor of the river package if the coordinates of this array are not rearranged.